lib/dry/system/plugins/injector_mixin.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Dry
+  module System
+    module Plugins
+      # @api private
+      class InjectorMixin < Module
+        MODULE_SEPARATOR = "::"
+
+        attr_reader :name
+
+        def initialize(name: "Deps")
+          @name = name
+        end
```

Comment:
Rubocop is saying:

```
Lint/MissingSuper: Call `super` to initialize state of the parent class.
```

Do we really think we need to do this here?

Reply:
I have a feeling we should always delegate to `super`. Who knows where this class can end up being used later? If it doesn't properly delegate, this would break expectations.

Reply:
My question is more about whether it's really necessary when inheriting from `Module`. We already don't do it across the other instances of `Module` inheritance in this codebase, and it's not something that's appeared necessary to me when using this technique elsewhere.

Reply:
@timriley I doubt it plays a role in this particular case. It's not a problem to have `super` calls to quell rubocop, though

Reply:
I think it's beneficial to have a habit of calling super, even if it's a noop. It may happen that you really need to call super, but you don't have this habit and you end up debugging why things don't work just realize you forgot to call super. Been there done that 🙂